specification/compute/resource-manager/Microsoft.Compute/stable/2021-07-01/compute.json

```diff
@@ -5135,7 +5140,12 @@
             }
           }
         },
-        "x-ms-long-running-operation": true
+        "x-ms-long-running-operation": true,
+        "x-ms-examples": {
+          "Create a VMSS with an extension that has suppressFailures enabled": {
+            "$ref": "./examples/CreateAScaleSetWithExtensionsSuppressFailuresEnabled.json"
```

Comment:
update to "./examples/compute/CreateAScaleSetWithExtensionsSuppressFailuresEnabled.json"

Reply:
there isn't a compute dir under examples.